src/io/flutter/view/FlutterView.java

```diff
@@ -430,24 +430,29 @@ private void presentDevTools(FlutterApp app, InspectorService inspectorService,
     final DevToolsManager devToolsManager = DevToolsManager.getInstance(app.getProject());
 
     if (devToolsManager.hasInstalledDevTools()) {
-      addBrowserInspectorViewContent(app, inspectorService, toolWindow, isEmbedded);
+      AsyncUtils.invokeLater(() -> addBrowserInspectorViewContent(app, inspectorService, toolWindow, isEmbedded));
```

Comment:
shouldn't this method instead be asserting that we are on the UI thread? seems on to have it start on some random thread.

Reply:
Do you mean `addBrowserInspectorViewContent` should assert that we're on the UI thread? I can add that as well, but I think I have to use `invokeLater` here since `presentDevTools` isn't running on the UI thread.